src/diagrams/gantt/ganttDb.js

```diff
@@ -12,6 +12,7 @@ let includes = [];
 let excludes = [];
 let links = {};
 let title = '';
```

Comment:
I didn't end up adding the default here because this is one of the charts that renders its title as a visible element. There probably aren't a ton of gantt charts without titles in the wild, but I didn't want existing ones to suddenly display differently.

This does make me wonder, however, if the title will get read twice by an assistive device, since it is both a visible element _and_ a `title` tag?

Reply:
Ah - good call! Hmm I'd have to take a look at the markup. If it's in the DOM twice then most likely it will be read twice.

Reply:
I'm not really a pro at navigating with VoiceOver 😬 , but when loading `gantt.html` in the `demos` folder, and navigating to the `svg`, I hear 'in a gantt diagram ${description text}, image', which I think is what we expect?

Reply:
I took a look as well, and you are correct! That is what we want. Since we have `aria-labelledby="${title-id} ${description-id}"` and `role="image"` it overrides everything else inside of the SVG. So, even though the text is in the DOM twice, it won't read the second one (which looks like this, right before the closing `</svg>` tag `<text x="632" y="25" class="titleText">A Gantt Diagram</text>`)